src/world/Explosion.php

```diff
@@ -125,20 +120,17 @@ public function explodeA() : bool{
 							}
 
 							$state = $subChunk->getBlockStateId($vBlockX & SubChunk::COORD_MASK, $vBlockY & SubChunk::COORD_MASK, $vBlockZ & SubChunk::COORD_MASK);
+							if(in_array($blockFactory->fromStateId($state)->getTypeId(), $this->excludedBlockTypeIds, true)){
```

Comment:
Don't use in_array, it's slow.

Reply:
Now I use `isset`, from some tests `isset` is the fastest.